### ISSUE TITLE
A11y-NVDA When moving focus across tabs always calls out tab as selected

### ISSUE THREAD

**Issue by giltza** [PolymerElements/paper-tabs#176]
### Description
NVDA accesability tool announces each paper-tab that is given a focus that it is the selected tab.

paper-tab is an aria role 'tab' implementation and should adhere to this aria role spec at https://www.w3.org/TR/wai-aria-1.1/#tab

Quote from aria spec.
> In either case, authors should ensure that a selected tab has its aria-selected attribute set to true, that **inactive tab elements have their aria-selected attribute set to false**, and that the currently selected tab provides a visual indication that it is selected. In the absence of an aria-selected attribute on the current tab, user agents should indicate to assistive technologies through the platform accessibility API that the currently focused tab is selected.

iron-menu-behavior removes the aria-selected attribute rather then setting it to false.

### Expected outcome

NVDA announces each paper-tab we traverse as selected.

### Actual outcome

Only the selected tab should be announced as selected.

### Steps to reproduce

1) Enable NVDA accessibility tool, 
2) Go to paper-tabs demo
3) Using the Keyboard move the focus from one tab to another (While not changing the selected state)

### Browsers Affected
- [x] Chrome
- [x] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by JanMiksovsky** [comment]
This issue was moved to PolymerElements/iron-menu-behavior#75
